solv/tdnfpackage.c

```diff
@@ -820,6 +820,60 @@ SolvGetPackageId(
     goto cleanup;
 }
 
+uint32_t
+SolvGetPkgChecksumFromId(
+    PSolvSack pSack,
+    uint32_t dwPkgId,
+    int checksumtype,
```

Comment:
Maintain the variable naming generic through out the code
int checksumtype --> int checksumType
int checksumlen --> int checksumLen